osu.Game/Storyboards/StoryboardLayerVideo.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Game.Storyboards.Drawables;
+using osuTK;
+
+namespace osu.Game.Storyboards
+{
+    public class StoryboardLayerVideo : StoryboardLayer
+    {
+        public StoryboardLayerVideo(string name, int depth, bool masking)
```

Comment:
Naming feels backwards. `StoryboardVideoLayer`?

Reply:
That's how I had it, then i double-took. Will change back.